### DISCUSSION TITLE
Documentation about post-update authorization/access-control rules?

### DISCUSSION THREAD

**Discussion by franciscolourenco** [hasura/graphql-engine#6447] [Question] [answered]
Hello,

Is there any documentation about the post-update authorization/access-control rules?

Thanks!

**Comment by tirumaraiselvan** [comment, marked as answer]
The API has a `check` key (this is the condition for post-update) which is documented here: https://hasura.io/docs/1.0/graphql/core/api-reference/schema-metadata-api/permission.html#id10
